lib/pbench/server/indexer.py

```diff
@@ -447,8 +447,7 @@ def dump_templates(self):
         sys.stdout.flush()
 
     def update_templates(self, es, target_name=None):
-        """Push the various Elasticsearch index templates required by pbench.
-        """
+        """Push the various Elasticsearch index templates required by pbench."""
```

Comment:
This line might be the one that's too long for black's approval? Try reformatting manually with `black .` (which will check and fix any nonconforming file).

Reply:
Weirdly this reformat happened after I ran `Black indexer.py`

Reply:
What version of black are you running interactively? Various versions have sometimes incompatible ideas of what's "good", which is why I locked our lint version of black to 19.10b0.

Reply:
ahh good catch thanks @dbutenhof I have black version 20.8b1 inside the virtual env but had 19.10b0 outside venv, should have updated it inside venv, yikes